src/components/atoms/VolumeControl/VolumeControl.scss

```diff
@@ -0,0 +1,45 @@
+@import "scss/constants";
```

Comment:
extracted `VolumeControl` component out of `NavBar.tsx` was still using styles from `NavBar.scss`